Добавление проекта на ветке develop2

AnnBerinceva opened the pull request.

AnnBerinceva committed:
Добавление проекта на ветке develop2

mkouznetsov-git commented on src/main/java/UserClient.java:
```diff
@@ -0,0 +1,93 @@
+                 .then()
+                 .statusCode(202);
+     }
+     public void delete(String accessToken) {
```
Что это?

AnnBerinceva replied:
Исправила (удалила).

mkouznetsov-git commented on src/test/java/ChangeUserTest.java:
```diff
@@ -0,0 +1,56 @@
+public class ChangeUserTest {
+
+    UserClient userClient;
+    Token token;
```
Переменная не используется. Для чего она нужна?

AnnBerinceva replied:
Удалила.

mkouznetsov-git replied:
Не исправлено

AnnBerinceva replied:
теперь точно удалила)

mkouznetsov-git commented on src/test/java/CreateOrderTest.java:
```diff
@@ -0,0 +1,77 @@
+    @Before
+    public void init() {
+
+        Token token = new Token();
```
Лишняя переменная. Так же это нужно поправить в других классах.

AnnBerinceva replied:
Удалила.

AnnBerinceva committed:
Изменение метода удаления курьера, убрала лишние данные, новые результаты allure

AnnBerinceva committed:
изменение README

AnnBerinceva committed:
Delete allure-results directory

По ошибке подгрузила тяжелую директорию с результатами аллюр 
Проект сильно вис из-за нее

AnnBerinceva committed:
удаление переменной token
